dasharo-security/tpm-support.robot

```diff
@@ -121,15 +121,77 @@ TPM003.003 Check TPM Physical Presence Interface (Windows)
     ${out}=    Execute Command In Terminal    tpmtool getdeviceinformation
     Should Contain    ${out}    PPI Version: 1.3
 
-# TPM003.004 Change active PCR banks with TPM PPI (firmware)
-#    [Documentation]    This test aims to verify that the TPM Physical Presence
-#    ...    Interface is working properly in the firmware by changing active TPM PCR banks.
-#    Skip If    not ${TPM_SUPPORTED_VERSION} == None    TPM003.004 not supported
-#    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    TPM003.004 not supported
-# TODO: https://docs.dasharo.com/unified-test-documentation/dasharo-security/200-tpm-support/#tpm003004-change-active-pcr-banks-with-tpm-ppi-firmware
-
+TPM003.004 Change active PCR banks with TPM PPI (firmware)
+    [Documentation]    This test aims to verify that the TPM Physical Presence
+    ...    Interface is working properly in the firmware by changing active TPM PCR banks.
+    Skip If    not ${TPM_SUPPORTED_VERSION} == 2    TPM003.004 not supported    #maby this should be NONE
```

Comment:
Please remove the comment, robotidy isn't happy about it. TPM1.2 had only SHA1 bank (and didn't call it that because there was no need to differentiate), so it obviously can't be changed there.